Automattic/wp-calypso
# Trail Map: Grouped feature map for trail map

Related to #

## Proposed Changes

* Refactors the plan group map.
* Provides the new group structure on any copy variant

## Testing Instructions



* Try `/start/plans?flags=onboarding/trail-map-feature-grid` should show the new groups on the features grid
* Try `/start/plans?flags=onboarding/trail-map-feature-grid-copy` should show the new groups on the comparison grid

## Pre-merge Checklist
